samples/django.md

```diff
@@ -270,9 +272,9 @@ In this section, you set up the database connection for Django.
 
 ## More Compose documentation
 
-- [User guide](index.md)
-- [Installing Compose](install.md)
-- [Getting Started](gettingstarted.md)
-- [Command line reference](reference/index.md)
-- [Compose file reference](compose-file/index.md)
-- [Sample apps with Compose](samples-for-compose.md)
+- [Docker Compose overview](../compose/index.md)
+- [Install Docker Compose](../compose/install.md)
+- [Getting Started with Docker Compose](../compose/gettingstarted.md)
+- [Docker Compose Command line reference](../compose/reference/index.md)
```

Comment:
The Compose Command line reference page appears as a floating page as it is not linked from the toc.